sw/device/silicon_creator/lib/base/sec_mmio.c

```diff
@@ -7,8 +7,15 @@
 #include "sw/device/lib/base/memory.h"
 #include "sw/device/silicon_creator/lib/base/abs_mmio.h"
 
-// FIXME: Linker configuration.
+#if HOST_BUILD
```

Comment:
@mcy should I be using `OT_OFF_TARGET_TEST` instead here? I just realised that seems to be the way we are doing the detection in the silicon creator code.

Reply:
I'm not really aware of the difference unfortunately =(

I haven't been following shenanigans with the build system lately.